numpy/distutils/system_info.py

```diff
@@ -489,8 +495,9 @@ def parse_config_files(self):
                 self.cp.add_section(self.section)
 
     def calc_libraries_info(self):
-        libs = self.get_libraries()
-        dirs = self.get_lib_dirs()
+        libs   = self.get_libraries()
+        dirs   = self.get_lib_dirs()
```

Comment:
Ahh, sorry, I always do that in my own scripts (it makes me read lines
faster). I will correct those things.

I have never done a reST list. Will correct those as well.

Regarding the tests, feel free to ping me with questions if any arise. :)

2015-02-25 8:43 GMT+01:00 Ralf Gommers <notifications@github.com>:

> In numpy/distutils/system_info.py
> <https://github.com/numpy/numpy/pull/5597#discussion_r25322728>:
>
> > @@ -489,8 +495,9 @@ def parse_config_files(self):
> >                  self.cp.add_section(self.section)
> >
> >      def calc_libraries_info(self):
> > -        libs = self.get_libraries()
> > -        dirs = self.get_lib_dirs()
> > +        libs   = self.get_libraries()
> > +        dirs   = self.get_lib_dirs()
>
> can you undo the added spaces here? PEP8 frowns upon such = alignment
> between lines
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/numpy/numpy/pull/5597/files#r25322728>.
>



-- 
Kind regards Nick